python/instrumentation/openinference-instrumentation-beeai/pyproject.toml

```diff
@@ -23,20 +22,21 @@ classifiers = [
   "Programming Language :: Python :: 3.13",
 ]
 dependencies = [
-    "openinference-semantic-conventions>=0.1.17",
-    "opentelemetry-api>=1.31.1",
-    "opentelemetry-instrumentation>=0.52b1",
-    "opentelemetry-semantic-conventions>=0.52b1",
-    "openinference-instrumentation>=0.1.27",
-    "wrapt>=1.17.2",
+    "beeai-framework[search] (==0.1.32)",
```

Comment:
Probably best to nix this as a dependancy and make it work via the instruments block below.
```suggestion
```